libs/configurable-product/README.md

```diff
@@ -10,4 +10,4 @@ provides clear interfaces, models, and factories for the frontend of an ecommerc
 npm install @daffodil/configurable-product
 ```
 
-The `@daffodil/configurable-product` package is intended for use in conjuction with the `@daffodil/product` package. After installing the `@daffodil/product` package, import the `DaffProductMagentoDriverModule` and the `DaffConfigurableProductStateModule` into your app. These will add the `DAFF_CONFIGURABLE_PRODUCT_STORE_FEATURE_KEY` to the app root redux state along with configurable product state.
+The `@daffodil/configurable-product` package is intended for use in conjuction with the `@daffodil/product` package. After installing the `@daffodil/product` package, import the `DaffConfigurableProductMagentoDriverModule` and the `DaffConfigurableProductStateModule` into your app. These will add the `DAFF_CONFIGURABLE_PRODUCT_STORE_FEATURE_KEY` to the app root redux state along with configurable product state.
```

Comment:
I don't think the root README should be singling out the magento driver in particular.

Reply:
> Everything should be good now, though I need to rewrite these docs still.

yea I still need to fix this